devices/ledvance.js

```diff
@@ -58,6 +58,14 @@ module.exports = [
         extend: extend.ledvance.light_onoff_brightness_colortemp_color(),
         ota: ota.ledvance,
     },
+    {
+        zigbeeModel: ['Outdoor FLEX RGBW Z3'],
+        model: '4058075208360',
+        vendor: 'LEDVANCE',
+        description: 'SMART+ ZB OUTDOOR FLEX EU',
+        extend: extend.ledvance.light_onoff_brightness_colortemp_color(),
```

Comment:
Please provide the color temp range: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html#31-retrieving-color-temperature-range-only-required-for-lights-which-support-color-temperature

Reply:
sorry - range has been added